lib/src/parsers/phone_parser.dart

```diff
@@ -128,14 +128,45 @@ class PhoneParser {
   ///
   /// throws a PhoneNumberException if the country calling code is invalid
   @internal
-  static PhoneNumber fromRaw(String phoneNumber) {
+  static PhoneNumber fromRaw(String phoneNumber, { IsoCode? callerCountry, IsoCode? destinationCountry }) {
     phoneNumber = TextParser.normalize(phoneNumber);
-    phoneNumber = InternationalPrefixParser.removeInternationalPrefix(
-      phoneNumber,
-    );
-    final countryCode = CountryCodeParser.extractCountryCode(phoneNumber);
+    String withoutIntlPrefix;
+    if (callerCountry != null) {
+      // as we know the caller country, we can remove the prefix more exactly

```

Comment:
"more exactly" is not english. I'd say "as we know the caller country, we can remove the international prefix (exit code)".

Reply:
Fixed